### PR TITLE
Feature/efiling report

### PR DESCRIPTION
**Pull request by noahmanger**
This adds support for including `ResourcePages` as children of `DocumentFeedPages`, which allows something like the efiling study to be included in "Reports about the FEC".

image

The remaining to-do is to add the correct category for this report. Do you know what that should be @emileighoutlaw / @AmyKort ?

### CODE REVIEW

**Comment by noahmanger** [comment]
Added the "Study" category as an option for "Agency operations" reports so this is ready to go.

File: fec/home/templatetags/document_feed.py [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+from home.models import DocumentPage, ResourcePage
+from wagtail.wagtailcore.models import Page, Orderable
 
 register = template.Library()
 
+def get_documents(page, year='', category=''):
+    documents = DocumentPage.objects.child_of(page).live().order_by('-date')
+
+    if year:
+      documents = documents.filter(date__year=year)
```
Comment (xtine) [inline fec/home/templatetags/document_feed.py]: check indent

Comment (noahmanger) [inline fec/home/templatetags/document_feed.py]: Looks right to me?

Comment (xtine) [inline fec/home/templatetags/document_feed.py]: It looks like two spaces vs four spaces.

Comment (noahmanger) [inline fec/home/templatetags/document_feed.py]: Oh oops. Fixing!

**Comment by LindsayYoung** [comment]
Thank you for doing this!

**Pull request merged**